src/skills-builder/skills-builder-modal/SkillsBuilderModal.jsx

```diff
@@ -81,7 +109,7 @@ const SkillsBuilderModal = () => {
               {formatMessage(messages.goBackButton)}
             </Button>
             <Stepper.ActionRow.Spacer />
-            <Button onClick={onCloseHandle}>
+            <Button onClick={exitButtonHandle}>
```

Comment:
What does `esc` do as a keyboard shortcut? I would expect it to fire this action if we were in a modal but I don't think it's a real modal. It's ok if the answer is "it does nothing", I'm just curious.

Reply:
It fires [this function ](https://github.com/openedx/frontend-app-profile/blob/057c351fd0c484ce80481c70397e58609b71d59e/src/skills-builder/skills-builder-modal/SkillsBuilderModal.jsx#L64)

Reply:
ooh good point. From an accessibility standpoint, I'd imagine we want it to still close out, given we're treating this like a modal.

Reply:
Yes indeed! The `ModalDialog` component we are using in Paragon has that functionality built-in. It will fire its `onClose` call back function when the `esc` key is pressed or the `X` button clicked